frontend/src/libs/hooks/use-app-form/use-app-form.hook.ts

```diff
@@ -19,8 +20,10 @@ type Parameters<T extends FieldValues = FieldValues> = {
 };
 
 type ReturnValue<T extends FieldValues = FieldValues> = {
+  watch: UseFormWatch<T>;
   control: Control<T, null>;
   errors: FieldErrors<T>;
+  isValid: boolean;
```

Comment:
I think you used isValid to disable the button later on and that's it, shouldn't you name it isDisabled then? Or there is a reason to name it this way?

Reply:
It's a value from useForm react hook, so I didn't change it. May be in further uses in the button component it's better to name it isDisabled, but not there, at least it's an opposite thing)